src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx

```diff
@@ -109,14 +109,10 @@ function WorkspaceWorkflowsPage({policy, route}: WorkspaceWorkflowsPageProps) {
     }, [policy, route.params.policyID, availableMembers, usedApproverEmails]);
 
     const optionItems: ToggleSettingOptionRowProps[] = useMemo(() => {
-        const {accountNumber, addressName, bankName, bankAccountID} = policy?.achAccount ?? {};
+        const {addressName, bankName, bankAccountID} = policy?.achAccount ?? {};
         const shouldShowBankAccount = !!bankAccountID && policy?.reimbursementChoice === CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_YES;
         const bankIcon = getBankIcon({bankName: bankName as BankName, isCard: false, styles});
 
-        let bankDisplayName = bankName ?? addressName;
-        if (accountNumber && bankDisplayName !== accountNumber) {
-            bankDisplayName += ` ${accountNumber.slice(-5)}`;
-        }
```

Comment:
what is this change for?

Reply:
Yeah, sorry just doing some scope-creep cleanup of unused code